test/suite/test_rollback_to_stable10.py

```diff
@@ -306,9 +311,11 @@ def test_rollback_to_stable_prepare(self):
         self.assertEqual(keys_removed, 0)
         self.assertEqual(keys_restored, 0)
         self.assertGreaterEqual(upd_aborted, 0)
-        self.assertGreater(pages_visited, 0)
-        self.assertGreaterEqual(hs_removed, 0)
-        self.assertGreater(hs_sweep, 0)
+        # Since the prepared updates never made their way onto the page, we
```

Comment:
The main intention of this test is to check the extra updates that went into the history store but those are not present in the data store as part of the checkpoint when the checkpoint is in-progress. With this change, as it is not possible, do we need this test?

Reply:
Maybe, do you think we should? I think it's still valuable as a smoke test to race eviction and checkpoint and ensure that the system doesn't trip an assert or die in some other way.

Reply:
Yes, we can keep this test to ensure that we don't write anything more in the eviction when checkpoint is in-progress. 
Update the test description with the new intention of the test, so that it will be good.